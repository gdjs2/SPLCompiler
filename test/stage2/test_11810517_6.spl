struct st {
	int x;
};

struct st1 {
	struct st s1;
	int y;
};

struct st2 {
	struct st s2[10];
	int z;
};

int main() {
	struct st2 ss2;
	struct st1 ss1;
	ss2.s2[5] = ss1.s1;
	ss2.z = ss1.y;
	ss2.s2[8] = 5;
}