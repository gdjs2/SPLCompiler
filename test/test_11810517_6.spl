struct st {
	int f[10];
	int x;
};

int main() {
	struct st s;
	s.f[5] = 20;
}