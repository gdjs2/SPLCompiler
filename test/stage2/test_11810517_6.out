Error type 5 at Line 20: unmatching types appear at both sides of the assignment operator